### PR TITLE
Add http://*/* host permission to be able to access non-secure cookies

### PR DESCRIPTION
**Pull request by cmfcmf**
Without this host permission, the extension cannot access cookies not marked as "secure".

Bugs (internal): https://crbug.com/1497385, https://issuetracker.google.com/307706348

### CODE REVIEW

**Comment by cmfcmf** [comment]
Maksim, PTAL :)

**Comment by cmfcmf** [comment]
(this is pending the cookies team to confirm in crbug.com/1497385 that this is the appropriate way to fix this)

File: test/manifest.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "storage"
   ],
   "host_permissions": [
-    "https://*/*"
+    "https://*/*",
```
Comment (emaxx-google) [inline test/manifest.json]: Probably a single `"*://*/*"` pattern should work here as well? Not insisting though, if you'd prefer something that's more explicit and has already been tested by you.

**Review by emaxx-google — APPROVED** [review]
LGTM for the implementation. I've left one proposal, but it's optional. Thanks for the small cleanup in the tests as well.

File: test/manifest.json (hunk 1 above)
Comment (cmfcmf) [reply test/manifest.json]: Makes sense to me, changed!

File: src/manifest.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
     "storage"
   ],
   "host_permissions": [
-    "https://*/*"
+    "https://*/*",
```
Comment (emaxx-google) [inline src/manifest.json:14]: Hmm, you've left these two values here meanwhile the test's manifest is changed to `*://*/*`. I think both places should use the latter value.

Comment (cmfcmf) [reply src/manifest.json:14]: Hmm, I thought about changing it to `*://*/*` also outside of tests, but figured that maybe that might be overly broad, since it would also include `isolated-app:`. On the other hand, you could of course argue that anyone can easily write their own extension containing `*://*/*` - this feels similar to the filtering we had in this extension earlier for google.com cookies.
TL;DR: Don't really have a strong opinion on this, let me know what you'd prefer :)

Comment (emaxx-google) [reply src/manifest.json:14]: I based my suggestion on the documentation that claims that `*://` matches only `http` and `https`: https://developer.chrome.com/docs/extensions/mv3/match_patterns/ . Do you know whether the documentation is outdated? It doesn't mention `isolated-app` at all, neither for `*` nor as an explicit option.
